README.md

```diff
@@ -161,7 +161,7 @@ Enforce all the rules in this category, as well as all higher priority rules, wi
 |  | [require-prop-types](./docs/rules/require-prop-types.md) | require type definitions in props |
 | :wrench: | [v-bind-style](./docs/rules/v-bind-style.md) | enforce `v-bind` directive style |
 | :wrench: | [v-on-style](./docs/rules/v-on-style.md) | enforce `v-on` directive style |
-
+| :wrench: | [attribute-order](./docs/rules/attribute-order.md) | enforce ordering of attributes |
```

Comment:
Did you add it by hand? I see there is `fixable` set to null in rule, but I also see 🔧 here. Please run `npm run update` if you want to update readme or rule documentation header base on rule source.